### PR TITLE
Ajit_Assignment_1

### PR DESCRIPTION
**Pull request by AjitSJune20**
image

### CODE REVIEW

File: src/ajit/Assignment1.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package ajit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Assignment1 {
+
+	public static void main(String[] args) throws InterruptedException {
+		String os = System.getProperty("os.name").toLowerCase();
```
Comment (KrishnaKTechnocredits) [inline src/ajit/Assignment1.java:10]: I am happy that you got it.

File: src/ajit/Assignment1.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		driver.findElement(By.id("first name")).sendKeys("Ajit");
+		driver.findElement(By.id("last name")).sendKeys("Singh");
+		driver.findElement(By.id("knownlanguages")).sendKeys("Python");
+		driver.findElement(By.id("fresher")).click();
+		driver.findElement(By.id("java")).click();
+		driver.findElement(By.id("tw")).click();
+
+		// Reset the From
+		driver.findElement(By.id("resetBtn")).click();
+		if (driver.findElement(By.id("first name")).getAttribute("value").equals(""))
```
Comment (KrishnaKTechnocredits) [inline src/ajit/Assignment1.java:43]: here you forgot to add validation for radio button and checkbox.

File: src/ajit/Assignment1.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			System.out.println("From Reset Done Successfully");
+		else
+			System.out.println("From Reset  Unsuccessfully");
+
+		// Verify Page Redirect
+		driver.findElement(By.id("morePractice")).click();
+		Thread.sleep(5000);
+		System.out.println("Current URL : " + driver.getCurrentUrl());
```
Comment (KrishnaKTechnocredits) [inline src/ajit/Assignment1.java:51]: Where is the validation for current URL ?

File: src/ajit/Assignment1.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		System.out.println("AutomationByKrishna website open successfully.");
+		// Verify Page Title
+		String actualTitle = driver.getTitle();
+		String expectedTitle = "TECHNOCREDITS";
+		if (actualTitle.equals(expectedTitle))
+			System.out.println("Title Matched");
+		else
+			System.out.println("Title didn't match, Actual URL : " + actualTitle);
+		// Select Radio button
+		if (driver.findElement(By.id("maleG")).isSelected()) {
```
Comment (KrishnaKTechnocredits) [inline src/ajit/Assignment1.java:26]: for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.
You have done reverse validation.

Comment (AjitSJune20) [reply src/ajit/Assignment1.java]: Incorporated Changes.

File: src/ajit/Assignment1.java (hunk 3 above)
Comment (AjitSJune20) [reply src/ajit/Assignment1.java:58]: Incorporated Changes.

File: src/ajit/Assignment1.java (hunk 2 above)
Comment (AjitSJune20) [reply src/ajit/Assignment1.java]: Incorporated Changes.

**Pull request merged**